epub33/core/index.html

```diff
@@ -5345,11 +5345,12 @@ <h4>Deriving the Path Names of Files</h4>
 					<h4>URLs in the OCF Abstract Container</h4>
 
 					<p>The <a>container root URL</a> is the <a data-cite="url#concept-url">URL</a> [[URL]] of the
-						<a>Root Directory</a>. It is implementation specific, but MUST abide to the following requirements:</p>
+						<a>Root Directory</a>. It is implementation specific, but the implementation MUST have the
```

Comment:
I'm wondering if we could avoid the MUST here (which I remind is an RS conformance statement already defined in the RS spec) by saying something like:

> It is implementation specific, but in a conforming Reading Systems it will have the following properties:

Reply:
I think keeping the MUST in the content is something that makes clear to ebook designers that we have a strong requirement here. It does not harm keeping it; with our process, we may have shared tests for it anyway.  

(I have not checked, but I am almost sure we have similar duality with other features shared by the core and RS elsewhere...)

Reply:
On the one hand, I don't think it makes sense to have an RS MUST in the core spec.

  * The abstract says (informatively)
    > This specification defines the authoring requirements for EPUB Publications
  * The overview says (informatively)
    > This specification (…) includes the conformance requirements for EPUB Publications
  * And all our normative conformance criteria apply to an EPUB Publication or EPUB Creators.

On the other hand, I do agree that a MUST is somehow required, to make it possible to evaluate the URL-related conformance criteria.

What about changing the text to:
> It is implementation specific, but the EPUB Creators MUST assume it has the following properties:

Reply:
I like that. Have done it.